lecture/models.py

```
@@ -44,26 +44,27 @@
         related_name='pdf_files'
     )
     pdf_file = models.FileField(upload_to='lecture_pdfs/')
-    pdf_title= models.TextField(null=True, blank=True)
+    pdf_title = models.TextField(null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)